integration/docker/run_test.go

```diff
@@ -291,10 +295,21 @@ var _ = Describe("run", func() {
 	Context("stderr using run", func() {
 		It("should not display the output", func() {
 			args = []string{"--rm", "--name", id, Image, "sh", "-c", "ls /etc/foo"}
-			stdout, stderr, exitCode := dockerRun(args...)
+			stdout, stderr, exitCode = dockerRun(args...)
 			Expect(exitCode).To(Equal(1))
 			Expect(stdout).To(BeEmpty())
 			Expect(stderr).To(ContainSubstring("ls: /etc/foo: No such file or directory"))
 		})
 	})
+
+	Context("stdin using run", func() {
+		It("should not display the stdin", func() {
+			Skip("Issue https://github.com/clearcontainers/runtime/issues/932")
```

Comment:
@GabyCT I see that this issue is not reproducible on kata, so we can remove the skip.